src/components/SiteHeader.js

```diff
@@ -0,0 +1,78 @@
+import React, { Fragment, useEffect} from 'react';
+import { Link } from 'react-router-dom/cjs/react-router-dom.min';
+
+
+import Logo from 'assets/images/logo.svg';
+
+const SiteHeader = () => {
+    useEffect(()=> {
+        // Mobile menu Drawer
+        const toggleMenu = () => {
+          const menu = document.querySelector('.nav');
+          const overlay = document.querySelector('.overlay');
+    
+          menu.classList.toggle('is-active');
+          if(overlay!==null){
+              overlay.classList.toggle('is-active');    
+            }
+        };
+    
+        const buttons = document.querySelectorAll('.menu-toggle, .overlay, .menu-close');
+    
+        for (let i = 0; i < buttons.length; i++) {
+          buttons[i].addEventListener('click', toggleMenu);
+        }
+    
+        // Header Sticky
+        window.addEventListener('scroll', () => {
```

Comment:
can we remove event listeners too when the component unmounts? returning a function which does calls [removeEventListener](https://developer.mozilla.org/en-US/docs/Web/API/EventTarget/removeEventListener) should be sufficient.